### PR TITLE
Improvements for issues

### PR DESCRIPTION
**Pull request by KamilWo**
Fixed footer.
Fixed problems with register views.
Removed links for photostack images.
Changed css selector for smooth scrolling - fixed navigation
of slider in home page.
Buttons changed to links looking like button.
Some headers changed to paragraphs.
Changed css for photostack photo description.
Changed columns min-heights css for category images.
Changed tab navigation in about_us.xml page.
Changed headers font sizes.
Removed outlines for links which were appearing in Firefox.
Slider is appearing on small devices.
Changed source for logo images in footer.
Changed headings for smaller fonts sizes.
Smaller, responsive down_arrow_scroll and turned off animation.
Fixed button for resetting password confirmation.
Fixed fonts css for login/register/password reset views.

### CODE REVIEW

File: addons/bestja_frontend_ucw/static/src/css/footer.css [hunk 1]
```diff
@@ -57,49 +77,46 @@ footer{
...
+        margin-left: calc(50% - 97.5px);
+    }
 }
 
-.footer_address li{
-    color: #767676;
-    list-style: none;
-    padding-bottom: 12px;
+//As response to /* ----- BOOTSTRAP FIX ----- */ in /website/static/src/css/website.css
+//In footer there is need for column as in original bootstrap
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_ucw/static/src/css/footer.css]: This is not a proper syntax of CSS comment.

File: addons/bestja_frontend_ucw/static/src/css/custom.css [hunk 2]
```diff
@@ -113,40 +129,130 @@ a:hover, a:focus{
...
+h5, .h5{
+    font-size: 10px;
+}
+
+h6, .h6{
+    font-size: 8.5px;
+}
+
+@media (min-width: 768px) {
+    h1, .h1{
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_ucw/static/src/css/custom.css]: What’s the purpose of duplicating tag names as CSS classes?

File: addons/bestja_frontend_ucw/static/src/css/footer.css (hunk 1 above)
Comment (KamilWo) [inline addons/bestja_frontend_ucw/static/src/css/footer.css]: Syntax changed, thank you.

File: addons/bestja_frontend_ucw/static/src/css/custom.css (hunk 2 above)
Comment (KamilWo) [inline addons/bestja_frontend_ucw/static/src/css/custom.css]: Classes were removed, but they were taken from boostrap css library.

**Pull request merged**